frontend/src/components/products/CreateForm.jsx

```diff
@@ -38,19 +39,33 @@ const CreateProduct = () => {
     setImages([...event.target.files]);
   };
 
+  const handleFormSubmit = async (event) => {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append("category", values.category);
+    formData.append("name", values.name);
+    formData.append("estimateUnit", values.estimateUnit);
+    formData.append("priceEstimate", values.priceEstimate);
+    formData.append("description", values.description);
+    formData.append("contactNumber", values.contactNumber);
+    formData.append("locations", values.locations);

```

Comment:
[This](https://stackoverflow.com/questions/37673454/javascript-iterate-key-value-from-json) might help.

Reply:
Nice, changes made